src/SGB_Command_Packet.md

```diff
@@ -1,8 +1,7 @@
 # Command Packet Transfers
 
 Command packets (aka Register Files) are transferred from the Game Boy to
-the SNES by using P14 and P15 output lines of the JOYPAD register
-(FF00).  These same lines are also used to select the two rows in the
+the SNES by using P14 and P15 output lines of [the JOYPAD register](<#FF00 — P1/JOYP: Joypad>) (FF00).  These same lines are also used to select the two rows in the
```

Comment:
Maybe split this in two lines as the rest of the file?

Reply:
@avivace Not exactly sure what you mean, are you talking about the line length? It's pretty inconsistent in this file already, but I could at least harmonize it within this paragraph.

Reply:
Ups, you're right.